src/lib/Scenes/Artwork/Components/ArtworkActions.tsx

```diff
@@ -32,11 +32,32 @@ export class ArtworkActions extends React.Component<ArtworkActionsProps> {
     })
   }
 
+  handleArtworkShare = async () => {
+    const { title, artist: name, id } = this.props.artwork
+
+    try {
+      const result = await Share.share({
+        message: `${title} by ${name} on Artsy http://artsy.net/artwork/${id}`,
```

Comment:
This will only work for artworks with a single artist! We should check with @samchieng for how she wants to treat this case.

Reply:
Let's just show up to 3 artists max. I don't think it's as important for the user to see past that.

"[Artwork Title] by [Artist Name], [Artist Name], [Artist Name] on Artsy [URL]"